odtp/workflow.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, execution_data, working_path, secrets):
         self.image_names = []
         self.repo_urls = []
         self.commits = []
-        self.instance_names = []
+        self.container_name = []
```

Comment:
**fix**: Shouldn't it `self.container_names` here for consistency? Since it will be a list of container names